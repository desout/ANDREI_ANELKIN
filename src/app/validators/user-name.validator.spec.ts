import {TestBed} from '@angular/core/testing';
import {checkName} from './user-name.validator';
describe('UserNameValidator', () => {
  beforeEach(() => {
    TestBed.configureTestingModule({});
  });

  it('checkName: Should correctly check name', () => {
    const returnedValue = checkName('Drogba');
    const returnedValue2 = checkName('drogba');

    expect(returnedValue).toBe(true);
    expect(returnedValue2).toBe(false);
    expect(returnedValue2).toMatchSnapshot();
    expect(returnedValue).toMatchSnapshot();
  });
});
