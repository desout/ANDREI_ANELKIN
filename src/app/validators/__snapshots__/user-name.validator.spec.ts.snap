// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserNameValidator checkName: Should correctly check name 1`] = `false`;

exports[`UserNameValidator checkName: Should correctly check name 2`] = `true`;
